tests/test_core.py

```diff
@@ -37,454 +36,419 @@ def delete_if_arcpy_exists(data):
         arcpy.Delete_management(data)
 
 
-def get_test_gdb():
-    '''Returns a path to the fgdb for the calling test method.
-    '''
-    calling_method_name = inspect.stack()[1].function
+#: check for local sde
+HAS_LOCAL_SDE = arcpy.Exists(path.join(UPDATE_TESTS_SDE, 'ZipCodes'))
+def skip_if_no_local_sde():
+    if not HAS_LOCAL_SDE:
+        raise pytest.skip('No test SDE detected, skipping test')
 
-    return path.join(suite_data_folder, '{}.gdb'.format(calling_method_name))
 
+CHANGE_DETECTION = ChangeDetection([], 'blah')
 
 
-change_detection = ChangeDetection([], 'blah')
-class CoreTests(unittest.TestCase):
```

Comment:
**question**: what were the reasons to convert this from a class?

Reply:
Because you can't use the `test_gdb` fixture in a class.

Reply:
for what reasons?

Reply:
Because [I said so](https://github.com/agrc/forklift/commit/e3c5307d2d0c2e09cff34dc75baa65357021ee1d). ;)

Although, I can't find the docs where I got that info at the moment. I do know that I tried passing the fixture and it didn't work. [It looks like there may be a way to make it work](https://docs.pytest.org/en/5.4.3/fixture.html#using-fixtures-from-classes-modules-or-projects) but it might require calling `@pytest.mark.usefixtures` on each suite and I'm not sure how to get a reference to the value of `test_gdb`. I'm open to suggestions. Although, I think that I like the non-class tests better. `pytest` does seem to prefer stand-alone tests rather than classes to me.